Helpers/opent2t-translator-helper-hue/js/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "opent2t-translator-helper-hue",
```

Comment:
Since this is a new helper module, I would suggest not adding it.

---
In reply to: [80416298](https://github.com/openT2T/translators/pull/96#discussion_r80416298) [](ancestors = 80416298)